recipes/opene57/all/conanfile.py

```diff
@@ -0,0 +1,135 @@
+from conans import ConanFile, tools, CMake
+from conan.tools.microsoft import msvc_runtime_flag
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class Opene57Conan(ConanFile):
+    name = "opene57"
```

Comment:
opene57 or open57? Doesn't seem consistent currently: https://github.com/openE57/openE57

Reply:
actually the real name would be "openE57", but I was told that it's better to use lowercase

Reply:
indeed